content/en/docs/setup/additional-setup/config-profiles/index.md

```diff
@@ -30,32 +30,29 @@ your specific needs. The following built-in configuration profiles are currently
 
 1. **minimal**: the minimal set of components necessary to use Istio's [traffic management](/docs/tasks/traffic-management/) features.
 
-1. **sds**: similar to the **default** profile, but also enables Istio's [SDS (secret discovery service)](/docs/tasks/security/citadel-config/auth-sds).
-    This profile comes with additional authentication features enabled by default (Strict Mutual TLS).
-
 1. **remote**: used for configuring remote clusters of a
     [multicluster mesh](/docs/ops/deployment/deployment-models/#multiple-clusters) with a
     [shared control plane](/docs/setup/install/multicluster/shared-vpn/) configuration.
 
+1. **preview**: this profile enables experimental features that may not meet the same level of stability, performance, and security as the default profiles.
```

Comment:
```suggestion
1. **preview**: enables experimental features that lack the stability, performance, and security of features in the default profiles.
```

Reply:
nit: To me this makes it sound like they are all unstable, bad performance, and not secure. Do you agree? I think change `and` to `or` or `and/or` may make more sense. Let me know what you think